src/main/java/sg/ncl/MainController.java

```diff
@@ -138,6 +153,23 @@ public void futureplanDownload(HttpServletResponse response) {
         }
     }
 
+    @RequestMapping(value="/OrderForm_v1/download", method=RequestMethod.GET)
+    public void OrderForm_v1Download(HttpServletResponse response) {
+        response.setContentType("application/pdf");
+        try {
+            File fileToDownload = new File("src/main/resources/downloads/OrderForm_v1.pdf");
+            InputStream is = new FileInputStream(fileToDownload);
+            response.setContentType("application/force-download");
+            response.setHeader("Content-Disposition", "attachment; filename=OrderForm_v1.pdf");
+            IOUtils.copy(is, response.getOutputStream());
+            response.flushBuffer();
+            is.close();
+        } catch (IOException ex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)